backend/src/routes/bookings.js

```
@@ -0,0 +1,166 @@
+                jobUpdates.status = 'in_progress';
+                break;
+
+            case 'complete':
+                if (profile.role !== 'worker' && profile.role !== 'admin') return res.status(403).json({ success: false, error: 'Only worker can complete' });
+                updates.status = 'completed';
+                updates.actual_end = new Date().toISOString();
+                jobUpdates.status = 'completed';
+                break;
+
+            case 'cancel':
+                const cancelledStatus = profile.role === 'customer' ? 'cancelled_by_customer' : 'cancelled_by_worker';
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)